### ISSUE TITLE
AVT 1 - Tag has DAP violation when disabled

### ISSUE THREAD

**Issue by snidersd** [carbon-design-system/carbon#4539]
## Environment
macOS Mojave version 10.14.6
Chrome Version Version 77.0.3865.90
Carbon v10 - React
DAP IBM Accessibility 1.8.0.0 Sept 2019 Ruleset

Detailed Description
Go to the React Component and select the disable knob
Run DAP and the following violations are present (see screenshot):
<img width="951" alt="Screen Shot 2019-11-01 at 4 23 16 PM" src="https://user-images.githubusercontent.com/21676914/68057503-1bb2c100-fcc4-11e9-9107-9adbd4d886ab.png">

**Comment by dakahn** [comment]
As discussed offline disabled components are exempt from 1.4.3's contrast requirements. Looks like an error with DAP 👍
